tests/benchmark.py

```diff
@@ -0,0 +1,40 @@
+"""
+BENCHMARK
+
```

Comment:
I'd prefer that this file was called something like benchmarkmodel or timemodel so that it's clear from the name what it is meant for. There could be other things that we'd want to benchmark.